Vagrantfile

```diff
@@ -0,0 +1,44 @@
+# -*- mode: ruby -*-
```

Comment:
Also, it maps your source from that local directory to a directory called /vagrant/ within the virtual machine. I actually symlink it to /home/vagrant/chesster